torch/__init__.py

```diff
@@ -36,7 +36,7 @@
 
 
 if TYPE_CHECKING:
-    from .types import Device, IntLikeType
+    from .types import Device, IntLikeType, _bool, _int, _float, FloatLikeType, _str, BoolLikeType
```

Comment:
Importing `_bool`, etc to be consistent with `torch/_C/_init.pyi`.

It may just be simpler to use `builtins.bool` etc.